### ISSUE TITLE
metrowindow topmost property binding issue

### ISSUE THREAD

**Issue by sg2707** [MahApps/MahApps.Metro#3738]
I am using MahApps for my WPF application and when the metro window is used, the TopMost property is not able to bind. If i set it as a static value it works though.

```
<Controls:MetroWindow x:Class="SabreContractSearch.Views.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:prism="http://prismlibrary.com/"
        xmlns:Controls="clr-namespace:MahApps.Metro.Controls;assembly=MahApps.Metro"
        xmlns:iconPacks="http://metro.mahapps.com/winfx/xaml/iconpacks"
        prism:ViewModelLocator.AutoWireViewModel="True"
        Topmost= "{Binding Pin}" Title="{Binding Title}" Height="350" Width="525" Name="FrmMainWindow" >
```
Pin is a boolean property here.

If i remove the metro window and use the normal window tag for WPF, it works. Is this a bug?

**Comment by punker76** [comment]
@sg2707 This should be working in v1.6.5 and current source. The main demo has a Binding to this property via Menu Item. So maybe you can create or upload this simple sample here on GitHub? Thx.

![image](https://user-images.githubusercontent.com/658431/74415084-f5fc9280-4e42-11ea-8864-a1a70f8077be.png)

**Comment by sg2707** [comment]
In this example it is binding the menu item to the Window property, What I am trying is to bind the viewmodel property to the window property.

` Topmost= "{Binding Pin}"`

**Comment by timunie** [comment]
Hi @sg2707 
I think there should be no difference between Binding to `MenuItem` and `ViewModel`. It would be useful to have a look into your App or a sample App. I think there is an issue in the MVVM. 

Happy coding,
Tim

**Comment by sg2707** [comment]
As i mentioned in the issue, if i change the metro window to WPF window, there is no issue. same code works. How am i supposed to give the app / sample app here. seriously, i'm new to github. sorry!! :(

**Comment by timunie** [comment]
@sg2707 I can confirm the issue with your repro.

**Comment by punker76** [comment]
@sg2707 I found the bug. It happens at WindowChromeBehavior from ControlzEx. The Binding will be broken by our Topmost hack. I will fix this.

**Comment by punker76** [comment]
@sg2707 It's fixed now and available with [ControlzEx v4.2.2](https://github.com/ControlzEx/ControlzEx/releases)
